root/static/scripts/relationship-editor/common/entity.js

```diff
@@ -6,6 +6,18 @@
 var mergeDates = require('./mergeDates.js');
 var deferFocus = require('../../edit/utility/deferFocus.js');
 
+function getDirection(relationship, source) {
+  let entities = relationship.entities();
+
+  if (source === entities[0]) {
+    return 'forward';
+  }
+
+  if (source === entities[1]) {
+    return 'backward';
+  }
+}
```

Comment:
This will return `undefined` if `source` is not used by the relationship, which should never happen, but if it does I'd rather the function not return an untrue value, hence the explicit checks.